arduino/smartcar/smartcar.ino

```diff
@@ -1,4 +1,11 @@
 #include <Smartcar.h>
+#include <vector>
```

Comment:
Arduino IDE cannot find the vector file

Reply:
> Arduino IDE cannot find the vector file

Sure it can. If you are compiling for the correct board (ESP32) of course.

Reply:
Yes, it's ok now, thanks